### ISSUE TITLE
[BUG]: raw_score=True dees not work in the latest version of Pycaret for binary classification

### ISSUE THREAD

**Issue by Prashavu** [pycaret/pycaret#2869]
### pycaret version checks

- [X] I have checked that this issue has not already been reported [here](https://github.com/pycaret/pycaret/issues).

- [X] I have confirmed this bug exists on the [latest version](https://github.com/pycaret/pycaret/releases) of pycaret.

- [ ] I have confirmed this bug exists on the master branch of pycaret (pip install -U git+https://github.com/pycaret/pycaret.git@master).

### Issue Description

I am trying to get the probability of conversion for a binary classification problem.
raw_score=True does not work !

### Reproducible Example

```python
from pycaret.datasets import get_data
juice = get_data('juice')
from pycaret.classification import *
exp_name = setup(data = juice,  target = 'Purchase')
lr = create_model('lr')
pred_holdout = predict_model(lr)
pred_unseen = predict_model(lr, data = unseen_dataframe,raw_score=True)
```

### Expected Behavior

Get two column in the output dataframe pred_unseen `Score_0` amd `Score_1`

### Actual Results

```python-traceback
Not getting any probability column.
```

### Installed Versions

<details>
3.0.0
</details>

**Comment by ngupta23** [comment]
If you are using the `pycaret-ts-alpha` install, that only works for time series and has been deprecated now. Please switch to this install as it now has all modules integrated together

`pip install --pre pycaret`

**Comment by Prashavu** [comment]
Im using the same. Version 3.0.0.rc3. And this was working for 2.3.10 s few days ago

**Comment by celestinoxp** [comment]
I found possible problem. you use python 3.7 and the problem could come from scikit-learn.

You must create a new environment with python 3.8 or python 3.9.

**Comment by celestinoxp** [comment]
> Estou usando o mesmo. Versão 3.0.0.rc3. E isso estava funcionando por 2.3.10 s alguns dias atrás. Estou pretendendo usá-lo para problemas de classificação binária

What version of scikit-learn you have?

**Comment by Prashavu** [comment]
@celestinoxp sorry but I do not understand that language so I could not follow your previous sentence.

**Comment by celestinoxp** [comment]
> @celestinoxp sorry but I do not understand that language so I could not follow your previous sentence.

Please paste the output of `from pycaret import show_versions; show_versions()`

**Comment by tvdboom** [comment]
This a known error (see #2813) and has been fixed in #2866. It will be available with next release.
